librabbitmq/amqp_socket.c

```diff
@@ -989,51 +994,72 @@ int amqp_simple_wait_frame_noblock(amqp_connection_state_t state,
   }
 }
 
-static int amqp_simple_wait_method_list(amqp_connection_state_t state,
-                                        amqp_channel_t expected_channel,
-                                        amqp_method_number_t *expected_methods,
-                                        amqp_method_t *output) {
+static int amqp_simple_wait_method_list_noblock(amqp_connection_state_t state,
+                                                amqp_channel_t expected_channel,
+                                                amqp_method_number_t *expected_methods,
```

Comment:
I see that in this file there are a lot of lines which are longer then 80 symbols.
Should I correct them (not only lines changed by myself)?